### ISSUE TITLE
Command line use

### ISSUE THREAD

**Issue by dpk** [bbloomf/jgabc#56]
It would be nice if there were command-line versions of the tools, so one could do something like

```shell
gabctranscribe words.txt music.gabc > words-and-music.gabc
```

and produce combined GABC files in batch that way. The psalm tone tool would likely be somewhat tricky to set up this way given its large number of configuration options, but the GABC transcription tool is quite simple by comparison. It would ‘just’ be a matter of pulling out the actual code which integrates words and music together into a separate module which could be used both by the web tool online, and by a Node.js-powered command line tool.

**Comment by bbloomf** [comment]
Yes, this would be great.  Unfortunately I don't have time to work on this right now.  The whole codebase should really be refactored, and everything related to combining texts with GABC could go into its own project, which could have a command line interface.
